src/content/pages/account/signup/index.tsx

```diff
@@ -129,9 +130,35 @@ function SignupForm() {
 
         // Don't continue to signup logic if there are errors.
         // We cannot check the useState values, as these will not be updated until the next render.
-        if (fullNameInvalid || usernameInvalid || emailInvalid || passwordInvalid || confirmPasswordInvalid) return
+       if (fullNameInvalid || usernameInvalid || emailInvalid || passwordInvalid || confirmPasswordInvalid) return
+       
+        
 
         // TODO: Signup logic.
+        const asyncSignupCall = async () => {
+            try {
+                const response = await fetch('http://localhost:8000/raw-data/create-user', {
+                method: 'POST',
+                headers: {
+                    'Content-Type': 'application/json',
+                  },
+                body: JSON.stringify({
+                    email: email,
```

Comment:
`email: email,` can be written as `email,`

Reply:
Done.